Converter/src/indexer.cpp

```diff
@@ -1162,9 +1162,9 @@ SoA toStructOfArrays(Node* node, Attributes attributes) {
 			};
 			vector<P> ps;
 			P min;
-			min.x = std::numeric_limits<int64_t>::max();
-			min.y = std::numeric_limits<int64_t>::max();
-			min.z = std::numeric_limits<int64_t>::max();
+			min.x = 0;
+			min.y = 0;
+			min.z = 0;

```

Comment:
To be honest this was trial and error.

I had tried what you suggested but then the whole result got scrambled. Like the higher the node level the more points were pulled towards the min xy of the complete dataset. Maybe because the viewer build upon this bug in decoding the Morton code.

Then I tried setting it 0 and the problem of coordinate differences got a little bit less.

The true solution was all the rounding that I introduced in stead of casting to int32_t.

Reply:
The original initialization appears to be doing this:
    min = { -1, -1, -1 };
...as the result from int64_t max value conversions to int32_t.

Reply:
Perhaps this min struct could just be a constant, or totally eliminated. Point X, Y, Z values are relative to octree bounding box min, and always >= 0, are they not?

Reply:
Yes I think you're right. Just left it in there as Markus might want to throw in some other min values. But until now it was -1, -1, -1 indeed.